src/Projections/Windows/ComInteropHelpers.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using Windows.ApplicationModel.DataTransfer.DragDrop.Core;
+using Windows.Foundation;
+using Windows.Graphics.Printing;
+using Windows.Graphics.Printing3D;
+using Windows.Media.PlayTo;
+using Windows.Security.Credentials.UI;
+using Windows.UI.ApplicationSettings;
+using Windows.UI.ViewManagement;
+using WinRT;
+using Windows.UI.Input;
+using Windows.UI.Input.Core;
+using Windows.UI.Input.Spatial;
+using Windows.Media;
+using Windows.Security.Authentication.Web.Core;
+using Windows.Security.Credentials;
+using System.Runtime.InteropServices;
+
+namespace Windows.Interop
```

Comment:
Since none of this is in the system itself, I think this is "Microsoft.Windows.Interop. "Windows" is reserved namespace only for system components.

Reply:
Yes, given that interops are tightly coupled to WinRT classes, they could be colocated under a relative interop namespace.  But Kenny's comment about using the Microsoft.Windows.SDK.Win32Metadata is a great idea and sidesteps the namespace question.  That coupled with the extension method suggestion (which is great), should make namespaces mostly irrelevant.

Reply:
Updates to the PR define the interop classes in the same namespace as the main object, e.g. `Windows.UI.ApplicationSettings.AccountsSettingsPaneInterop`